### PR TITLE
Developer-sasanka    created bodime details update method

### PR DESCRIPTION
**Pull request by sasanka2023**
created bodime details update method and updated bodime mapper

### CODE REVIEW

File: webApp-backend/src/main/java/org/ApeBodima/webApp_backend/config/security/SecurityConfig.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         http
 
                 .authorizeHttpRequests((request) -> request
-                        .requestMatchers("/api/v*/registration/user").permitAll()
+                        .requestMatchers("/**").permitAll()
```
Comment (SPradeepIndie) [inline webApp-backend/src/main/java/org/ApeBodima/webApp_backend/config/security/SecurityConfig.java:38]: you have changed the security configuration
**Please rollback it to first state**
it is essential

**Review by SPradeepIndie — CHANGES_REQUESTED** [review]
I want a change the SecurityConfig file change ASAP

File: webApp-backend/src/main/java/org/ApeBodima/webApp_backend/config/security/SecurityConfig.java (hunk 1 above)
Comment (sasanka2023) [reply webApp-backend/src/main/java/org/ApeBodima/webApp_backend/config/security/SecurityConfig.java:38]: ok

**Review by SPradeepIndie — APPROVED** [review]
looks good

**Pull request merged**